BoxOffice/HomeView/ListCell.swift

```diff
@@ -0,0 +1,216 @@
+//
+//  ListCell.swift
+//  BoxOffice
+//
+//  Created by 이원빈 on 2023/01/03.
+//
+
+import UIKit
+
+final class ListCell: UICollectionViewCell {
+    
+    private let posterImageView: UIImageView = {
+        let imageView = UIImageView()
+        imageView.translatesAutoresizingMaskIntoConstraints = false
+        imageView.contentMode = .scaleAspectFit
+        imageView.tintColor = .systemGray3
+        
+        imageView.clipsToBounds = true
+        return imageView
+    }()
+    
+    private let mainStackView: UIStackView = {
+        let stackView = UIStackView()
+        stackView.translatesAutoresizingMaskIntoConstraints = false
+        stackView.axis = .horizontal
+        stackView.isLayoutMarginsRelativeArrangement = true
+        stackView.layoutMargins = UIEdgeInsets(top: 10, left: 0, bottom: 10, right: 0)
+        stackView.alignment = .top
+
+        stackView.spacing = 10
+        return stackView
+    }()
```

Comment:
한 줄씩 개행은 의도이신건가요? 아니면 없애주세요!

```suggestion
    private let posterImageView: UIImageView = {
        let imageView = UIImageView()
        imageView.translatesAutoresizingMaskIntoConstraints = false
        imageView.contentMode = .scaleAspectFit
        imageView.tintColor = .systemGray3
        imageView.clipsToBounds = true
        return imageView
    }()
    
    private let mainStackView: UIStackView = {
        let stackView = UIStackView()
        stackView.translatesAutoresizingMaskIntoConstraints = false
        stackView.axis = .horizontal
        stackView.isLayoutMarginsRelativeArrangement = true
        stackView.layoutMargins = UIEdgeInsets(top: 10, left: 0, bottom: 10, right: 0)
        stackView.alignment = .top
        stackView.spacing = 10
        return stackView
    }()
```

Reply:
주석이 있던 자리인 것 같네요! 지우고 그대로 두었군요.. 🧐